tests/client/amo/components/TestSearchSort.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import {
+  Simulate,
+  findRenderedComponentWithType,
+  renderIntoDocument,
+} from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import I18nProvider from 'core/i18n/Provider';
+import SearchSort from 'amo/components/SearchSort';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+function render(props) {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+  return findRenderedComponentWithType(renderIntoDocument(
+    <Provider store={createStore(initialState)}>
+      <I18nProvider i18n={getFakeI18nInst()}>
+        <SearchSort {...props} />
+      </I18nProvider>
+    </Provider>
+  ), SearchSort).getWrappedInstance();
+}
+
+describe('<SearchSortBase />', () => {
+  it('toggles the search sort options when clicking "sort" link', () => {
+    const root = render({ filters: { query: 'test' }, pathname: '/search/' });
+    const rootNode = findDOMNode(root);
+
+    assert.notInclude(rootNode.className, 'SearchSort--visible');
+
+    Simulate.click(root.searchToggle);
+
+    assert.include(rootNode.className, 'SearchSort--visible');
+  });
+
+  it('blurs the search sort toggle when closing the sort options', () => {
+    const fakeSearchToggle = { blur: sinon.stub() };
+    const root = render({ filters: { query: 'test' }, pathname: '/search/' });
+
+    // Open the sort options.
+    root.toggleSort();
+
+    sinon.stub(root, 'searchToggle', fakeSearchToggle);
```

Comment:
Looks like you could stub this before the first `root.toggleSort()` and check it wasn't called on open then check it was called on close. Not a bit deal but should shorten the test a little.